lib/Mojolicious/Plugin/Gravatar.pm

```diff
@@ -0,0 +1,53 @@
+package Mojolicious::Plugin::Gravatar;
```

Comment:
A bit more work than you'd *like* to do I'm sure, but, again, this is part of the process and this will be yet again another fantastic experience for you!

You should extract the M::P::Gravatar module that you built, this module / plugin should and can stand on it's own two feet.  The whole point of a module is to make it shareable / reuseable.  Right now, this module is not reusable, it's accessible via the Ama application only.  If you extract M::P::Gravatar into its own module and publish, you'll be able to use it again again from other apps and other people will be able to use it, too!!

I use App::git::ship to manage my packages.  This will create and manage all the necessary boilerplate code like `Makefile`s and other necessary files.  It will also manage package versions.  It will also publish your package to CPAN!

It would be a really great experience for you to do that, to release something to *the world*!  And this very simple module is the perfect case for which to do your first!